alyciacan/rancid-tomatillos
# Feature/error handling

#### What's this PR do?

- adds error handling

#### Where should the reviewer start?

- error component directory and App.js

#### How should this be manually tested?

- pull down and check in the browser (Note: will need to change url in apiCalls to trigger error)

#### Any background context you want to provide?

- none

#### Are there any issues?

- not that I'm aware of

#### Next steps:

- create search form component
- add loading data placeholder

#### Questions:

- none
